detox/src/devices/android/Emulator.js

```diff
@@ -93,9 +102,9 @@ class Emulator {
         default:
           return 'auto';
       }
-    } else {
-      return 'host';
     }
+
+    return undefined;
```

Comment:
@RyanThomas73 for backwards compatibility, please return `host` as the default.

Reply:
@d4vidi Returning host as the default reintroduces the problem where it will always include `-gpu host` with the emulator command preventing the emulator command from honoring an explicit gpu mode specific in the emulator config via `hw.gpu.mode=`

Reply:
We could choose a specific value to map to `undefined` so it uses the emulator config default (e.g. `detox test .... --gpu default`) but I feel its better the way it is in the PR now, the user shouldn't have to explicitly specify default to get the default behavior from the emulator.

Reply:
Yes, you are right - I was not aware of that.
Nevertheless the issue of back-compatibility still _is_ concerning (keeping the community happy). That being said, I take it that the crucial use cases are of the CI, where typically emulators are run `headless`, which isn't affected in this case. So, bottom line here - I'll approve and merge.
Thank you :)